### DISCUSSION TITLE
total income displayed in the 'Farm' page  less than  the total income in the incoming history displayed in the 'Wallets' page

### DISCUSSION THREAD

**Discussion by yyqqwang** [Chia-Network/chia-blockchain#5012] [Support]
Hi  all，

    Today  I found my total income (XCH Total Chia Farmed) displayed in the 'Farm' page  is  24 XCH ,  but the total income in the incoming history displayed in the 'Wallets' page adds up to  27 XCH .   I  checked the incoming history then I found that  on 05/14/2021  11:04 am , I got 1.5 XCH incoming ， and on 05/14/2021  11:02 am  I got 1.5 XCH incoming，and it seems that  these 3  XCH are not added up to the total income.     While  the full node and the wallet are connected and synced.
            I  have no idea how to resolve this disorder. Hope some can help.
           The OS  version  is window 10, and the Chia full node version is 1.1.5

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
